apps/andi/mix.exs

```diff
@@ -46,7 +46,7 @@ defmodule Andi.MixProject do
       {:sobelow, "~> 0.8", only: :dev},
       {:phoenix, "~> 1.4"},
       {:phoenix_html, "~> 2.13"},
-      {:phoenix_live_reload, "~> 1.2", only: :dev},
+      {:phoenix_live_reload, "~> 1.2", only: [:dev, :integration]},
```

Comment:
Didn't we remove this requirement from this first card?  Is there something in this view that requires live view?

Reply:
I'd like to know the answer to this as well; I would appreciate a documented response in case others are also curious.

Reply:
Live reload is a dev tool, LiveView is something different.

https://github.com/phoenixframework/phoenix_live_reload